packages/metrics/package.json

```diff
@@ -13,7 +13,9 @@
     "commit": "commit",
     "test": "npm run test:unit",
     "test:unit": "jest --group=unit --detectOpenHandles --coverage --verbose",
-    "test:e2e": "jest --group=e2e",
+    "test:e2e:nodejs12x": "RUNTIME=nodejs12x jest --group=e2e/logger",
```

Comment:
wrong copy/paste. And I don't think you need to specify the module after e2e since it's a module specific package.json so it won't see otheres e2e tests anyway

Reply:
Good catch!

Reply:
Good catch!